docs/source/cookbook.rst

```diff
@@ -199,5 +199,11 @@ FIXME
 Using a Different GPG Base Directory
 ====================================
 
-FIXME
+PyGPGME honours the ``GNUPGHOME`` environment variable. This means you can change the GPG Base Directory simply by setting it using ``os.environ``::
+
+    import os
+    import gpgme
+
+    os.environ['GNUPGHOME'] = '/path/to/dir'
+    ctx = gpgme.Context()
```

Comment:
If you're using more than one GPG homedir this will change for all dirs. Instead it should be noted that you can use:

ctx = gpgme.Context()
ctx.set_engine_info(<protocol>, <file_name>, <gpg_home_dir>)

This will change the GPG Base dir only for this context.